pkg/geo/geomfn/voronoi_test.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package geomfn
+
+import (
+	"github.com/stretchr/testify/require"
+	"testing"
+
+	"github.com/cockroachdb/cockroach/pkg/geo"
+)
+
+func TestVoronoiPolygons(t *testing.T) {
+
+	type args struct {
+		a geo.Geometry
+		tol float64
+		poly int
+	}
+	tests := []struct {
+		name       string
+		args      args
+		expected   geo.Geometry
+		expectedErr error
+	}{
+		{
+			name: "Computes Voronoi Polygons for a given MultiPoint",
+			args: args{
+				a: geo.MustParseGeometry("POINT(10 20)"),
+				tol: 0,
+				poly: 1,
+			},
+			expected: geo.MustParseGeometry("MULTILINESTRING EMPTY"),
+		},
+		//{
+		//	name: "Computes Voronoi Polygons for a given MultiPoint",
+		//	args: args{
+		//		a: geo.MustParseGeometry("MULTIPOINT ((280 300), (420 330), (380 230), (320 160))"),
+		//		tol: 0,
+		//		poly: 0,
+		//	},
+		//	expected: geo.MustParseGeometry("GEOMETRYCOLLECTION (POLYGON ((110 175.71428571428572, 110 500, 310.35714285714283 500, 353.515625 298.59375, 306.875 231.96428571428572, 110 175.71428571428572)), POLYGON ((590 204, 590 -10, 589.1666666666666 -10, 306.875 231.96428571428572, 353.515625 298.59375, 590 204)), POLYGON ((110 -10, 110 175.71428571428572, 306.875 231.96428571428572, 589.1666666666666 -10, 110 -10)), POLYGON ((310.35714285714283 500, 590 500, 590 204, 353.515625 298.59375, 310.35714285714283 500)))"),
```

Comment:
@otan here this test case is failing, that's why I left it commented it for now

Reply:
I picked up these tests from `c-deps/geos/tests/unit/capi/GEOSVoronoiDiagramTest.cpp` 😅

Reply:
Also, I'll rename poly to `onlyEdges`

Reply:
where did you get this result from?
as far as i understand it, we should be using onlyEdges = 0 only when using `ST_VoronoiLines`.

Reply:
ah nvm, i got that wrong.